src/services/queue-service.js

```diff
@@ -1,10 +1,40 @@
 import { UserNetworkInstance } from './network';
+import { setKeyWithExpiryToLocalStore } from '../utilities/local-storage';
+import { log } from "../utilities/log";
 
+//const DEFAULT_POLLING_DELAY_IN_SECONDS = 600;
```

Comment:
This will be the new default set within the SDK, commented so we can release as is for now.